cmd/publishing-bot/main.go

```diff
@@ -46,6 +46,7 @@ func main() {
 	configFilePath := flag.String("config", "", "the config file in yaml format")
 	dryRun := flag.Bool("dry-run", false, "do not push anything to github")
 	tokenFile := flag.String("token-file", "", "the file with the github toke")
+	rulesFile := flag.String("rules-file", "", "the file with repository rules")
```

Comment:
How do you imagine this to be used? Should it point to the source repository worktree? How can we make sure that we always have master checked out? We could put a checkout command into the init script in the init container.

Reply:
@sttts i see two options:

1) Same as config (via configMap), same plumbings
2) Fetch from URL (iow. when you want to keep the rules in the repo). Not sure if fetch from URL is reasonable and we rather should look into the latest cloned sources for the source repo and have some standard path where this file lives.

Reply:
@sttts i guess for beginning, I would go just with configMap and we can improve based on how we want to manage the rules.